src/standalone/env.go

```diff
@@ -108,7 +108,11 @@ func (env *environment) addInstallerFlavor() error {
 	if err != nil {
 		return err
 	}
-	env.InstallerFlavor = flavor
+	if flavor == "" {
```

Comment:
checkEnvVar() returns an error in case the envVar is not set, so this code is not reachable and flavor is always set to something

Reply:
fixed it by making it an "optional" env, which is actually true